contracts/ExoticPositionalMarkets/ExoticPositionalMarketManager.sol

```diff
@@ -0,0 +1,739 @@
+
+import "@openzeppelin/contracts-4.4.1/proxy/Clones.sol";
+import "./ExoticPositionalFixedMarket.sol";
+import "./ExoticPositionalOpenBidMarket.sol";
+import "../interfaces/IThalesBonds.sol";
+import "../interfaces/IExoticPositionalTags.sol";
+import "../interfaces/IThalesOracleCouncil.sol";
+import "../interfaces/IExoticPositionalMarket.sol";
+import "../interfaces/IExoticRewards.sol";
+
+// internal
+import "../utils/proxy/solidity-0.8.0/ProxyReentrancyGuard.sol";
+import "../utils/proxy/solidity-0.8.0/ProxyOwned.sol";
+
+contract ExoticPositionalMarketManager is Initializable, ProxyOwned, PausableUpgradeable, ProxyReentrancyGuard {
+    using SafeMathUpgradeable for uint;
+
+    uint private constant backstopTimeoutDefault = 4 hours;
+    uint private constant fixedBondAmountDefault = 100 * 1e18;
+
+    uint public fixedBondAmount;
+    uint public backstopTimeout;
+    uint public minimumPositioningDuration;
+    uint public claimTimeoutDefaultPeriod;
+    uint public pDAOResolveTimePeriod;
+    uint public safeBoxPercentage;
+    uint public creatorPercentage;
+    uint public resolverPercentage;
+    uint public withdrawalPercentage;
+
+    uint public maxOracleCouncilMembers;
+    address public exoticMarketMastercopy;
+    address public oracleCouncilAddress;
+    address public safeBoxAddress;
+
+    address public thalesBonds;
+    address public paymentToken;
+    uint public maximumPositionsAllowed;
+    uint public disputePrice;
+
+    mapping(uint => address) public pauserAddress;
+    uint public pausersCount;
+    mapping(address => uint) public pauserIndex;
+
+    mapping(address => address) public creatorAddress;
+    mapping(address => address) public resolverAddress;
+
+    mapping(uint => address) public activeMarkets;
+    uint public numOfActiveMarkets;
+
+    address public tagsAddress;
+    uint public maxNumberOfTags;
+    uint public backstopTimeoutGeneral;
+    uint public safeBoxLowAmount;
+    uint public arbitraryRewardForDisputor;
+    address public theRundownConsumerAddress;
+    mapping(address => bool) public isChainLinkMarket;
+    address public marketDataAddress;
+    bool public creationRestrictedToOwner;
+    uint public minFixedTicketPrice;
+    uint public disputeStringLengthLimit;
+    mapping(address => bool) public cancelledByCreator;
+    address public exoticMarketOpenBidMastercopy;
+    address public exoticRewards;
+    uint public marketQuestionStringLimit;
+    uint public marketSourceStringLimit;
+    uint public marketPositionStringLimit;
+    bool public openBidAllowed;
+    uint public withdrawalTimePercentage;
+
+    function initialize(
+        address _owner,
+        uint _minimumPositioningDuration,
+        address _paymentToken
+    ) public initializer {
+        setOwner(_owner);
+        initNonReentrant();
+        minimumPositioningDuration = _minimumPositioningDuration;
+        backstopTimeout = backstopTimeoutDefault;
+        maximumPositionsAllowed = 5;
```

Comment:
why are all these not sent as params?

Reply:
they are, it's a leftover, I will remove this code